app/views/shared/dwp/_online.html.slim

```diff
@@ -0,0 +1,3 @@
+.row.pad-top-thirty-px
```

Comment:
I'm not sure if we want to display the green box there permanently, when the DWP check is available? It rings a bell that I overheard that sort of conversation some time ago, but I'm not sure about it.